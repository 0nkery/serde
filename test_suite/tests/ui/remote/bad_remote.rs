#[macro_use]
extern crate serde_derive;

mod remote {
    pub struct S {
        a: u8,
    }
}

#[derive(Serialize)]
#[serde(remote = "~~~")]
struct S {
    a: u8,
}

fn main() {}
