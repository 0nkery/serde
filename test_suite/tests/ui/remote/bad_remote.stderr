error: failed to parse path: "~~~"
  --> $DIR/bad_remote.rs:11:18
   |
11 | #[serde(remote = "~~~")]
   |                  ^^^^^

error: aborting due to previous error

